pkg/processor/sumologicschemaprocessor/testdata/config.yaml

```diff
@@ -9,7 +9,8 @@ processors:
     translate_attributes: false
   sumologic_schema/disabled-telegraf-attribute-translation:
     translate_telegraf_attributes: false
-
+  sumologic_schema/disabled-log-fields-convertion:
+    convert_attributes_to_fields: false
```

Comment:
I think that this name is too general - it suggests that this feature allows to converts all attributes to fields, but we only add severity number here. Also, we're converting a severity level into an attribute, so `attributes_to_fields` is misleading imo (if I'm not mistaken, fields in Sumo are the same thing as attributes in otlp, but correct me if I'm wrong)
I'd suggest something like `add_severity_level_attribute`

Reply:
true, that was my first idea, but then I decided to generalize that to maybe handle more attributes, however as it is now it make more sense to change as you suggest